google/resource_compute_instance.go

```diff
@@ -1142,3 +1235,79 @@ func resourceInstanceTags(d *schema.ResourceData) *compute.Tags {
 
 	return tags
 }
+
+func expandBootDisk(d *schema.ResourceData, config *Config, zone *compute.Zone, project string) (*compute.AttachedDisk, error) {
```

Comment:
It looks like we only use schema elements that are properties of `boot_disk.0`; we can reduce the information this function needs by passing in a limited set of data, like [BackendService's expandBackends()](https://github.com/terraform-providers/terraform-provider-google/blob/master/google/resource_compute_backend_service.go#L354)